### DISCUSSION TITLE
How powerful (i.e. how many clients) can Grapevine handle reliably?

### DISCUSSION THREAD

**Discussion by ScottTechMgr** [scottoffen/grapevine#99] [General]
We had great luck writing a web server application in Grapevine 4.2.2 but it was only used for a demo. Scaling up I have to wonder if Grapevine can handle 100....200....300 clients using the web service or if we need to rewrite using IIS for larger scale usage? Thanks in advance for your advice.

**Comment by scottoffen** [comment]
I have made every effort to ensure the library is performant, however I have not run any benchmarks on it. I would think the limitation really would be based on what `HttpListener` can handle, Grapevine itself is pretty lightweight.
